runtime/nvqir/qpp/QppCircuitSimulator.cpp

```diff
@@ -278,6 +282,12 @@ class QppCircuitSimulator : public nvqir::CircuitSimulatorBase<double> {
                         {state.data(), state.data() + state.size()}};
   }
 
+  bool isStateVectorSimulator() const override {
+    if constexpr (std::is_same_v<StateType, qpp::ket>)
+      return true;
+    return false;
```

Comment:
Just a question: we cannot do `return std::is_same_v<StateType, qpp::ket>;` ?

Reply:
Yes, thanks for the suggestion. Updated in 7999c2e80.